yt_dlp/extractor/roosterteeth.py

```diff
@@ -130,6 +129,30 @@ class RoosterTeethIE(RoosterTeethBaseIE):
             'release_date': '20141016',
         },
         'params': {'skip_download': True},
+    }, {
+        'url': 'https://roosterteeth.com/watch/rwby-bonus-21',
+        'info_dict': {
+            'id': '33',
+            'display_id': 'rwby-bonus-21',
+            'title': 'Volume 5 Yang Character Short',
+            'description': 'md5:8c2440bc763ea90c52cfe0a68093e1f7',
+            'episode': 'Volume 5 Yang Character Short',
+            'channel_id': '92f780eb-ebfe-4bf5-a3b5-c6ad5460a5f1',
+            'thumbnail': r're:^https?://.*\.(png|jpe?g)$',
+            'ext': 'mp4',
+            'availability': 'public',
+            'episode_id': 'f2a9f132-1fe2-44ad-8956-63d7c0267720',
+            'episode_number': 55,
+            'tags': None,
+            'season_id': None,
+            'season': None,
+            'series': 'RWBY',
+            'season_number': None,
```

Comment:
fields with `None` values can be excluded from the test dict
```suggestion
            'series': 'RWBY',
```

Reply:
on that note, is `None` for tags okay? I wasn't sure about that

Reply:
yeah that's ok